osf/external/messages/celery_publishers.py

```diff
@@ -0,0 +1,48 @@
+from kombu import Exchange
+from framework.celery_tasks import app as celery_app
+from framework.celery_tasks.handlers import enqueue_task
+
+
+def publish_deactivated_user(user):
+    enqueue_task(
```

Comment:
This had to use enqueue task to avoid blocking tests, I think it's good practice in case there's some temporary network issue.

Reply:
In theory, that could also be avoided with an `if settings.USE_CELERY:` conditional in `_publish_user_status_change`. I'm not sure the way it is now would be a problem, but does lead to more messages than necessary being queued.